packages/desktop-app/src/main.ts

```diff
@@ -385,6 +386,40 @@ const createMainWindow = () => {
     })
   })
 
+  bridge.on('whitelist-windows-defender', (nonDefaultFilePath?: string) => {
+    exec(
+      powershellWhitelistCommand,
+      {
+        env: {
+          WHITELIST_DIR: nonDefaultFilePath
+            ? path.join(nonDefaultFilePath, 'Salad/plugin-bin')
```

Comment:
```suggestion
            ? path.normalize(filePath)
```

We should take the full file path from the caller. This will give us more flexibility, and can be used in the very near future with Salad Bowl's new folder paths (which won't have `plugin-bin` in the path).